static/app/bootstrap/commonInitialization.tsx

```diff
@@ -3,15 +3,11 @@ import 'focus-visible';
 import {NODE_ENV} from 'sentry/constants';
 import ConfigStore from 'sentry/stores/configStore';
 import {Config} from 'sentry/types';
-import {setupColorScheme} from 'sentry/utils/matchMedia';
 
 export function commonInitialization(config: Config) {
   if (NODE_ENV === 'development') {
     import(/* webpackMode: "eager" */ 'sentry/utils/silence-react-unsafe-warnings');
   }
 
   ConfigStore.loadInitialData(config);
-
-  // setup darkmode + favicon
-  setupColorScheme();
```

Comment:
It's okay, we don't need to do it THIS early. Once the app mounts will be more than good enough